internal/installer/elasticagent_deb.go

```diff
@@ -112,7 +112,8 @@ func (i *elasticAgentDEBPackage) InstallCerts(ctx context.Context) error {
 
 // Logs prints logs of service
 func (i *elasticAgentDEBPackage) Logs() error {
-	return i.deploy.Logs(i.service)
+	// TODO we could read "/var/lib/elastic-agent/data/elastic-agent-*/logs/elastic-agent-json.log"
+	return systemCtlLog(context.Background(), "debian", i.Exec)
```

Comment:
```suggestion
	return systemCtlLog(context.Background(), "deb", i.Exec)
```

Reply:
Yeah, I though about that too, but we are consistently using "debian" across any span created in the DEB installer. I'd keep this as it's now, and send a PR updating all span names for debian, for consistency. Wdyt?